conf/app.ini

```diff
@@ -120,6 +120,8 @@ SSH_ROOT_PATH =
 SSH_KEY_TEST_PATH =
 ; Path to ssh-keygen, default is 'ssh-keygen' and let shell find out which one to call.
 SSH_KEYGEN_PATH = ssh-keygen
+; Disable SSH Authorized Key Backup when rewriting all keys, default is false
+SSH_DISABLE_AUTHORIZED_KEYS_BACKUP = true
```

Comment:
The value given here should be equal to the default value, so if the documentation is correct (default is false) the value should also be false. I see UPDATE_EXISTING has the same problem, but I don't expect you to fix it here, not being your code (would not be against doing it though)